main/src/ca/uwaterloo/flix/language/phase/typer/ConstraintGen.scala

```diff
@@ -717,24 +717,28 @@ object ConstraintGen {
         (resultTpe, resultEff)
 
       case Expr.TryWith(exp, effUse, rules, tvar, loc) =>
+        // try e with Eff { def f1(.., k1): r1 = e1, def f2(.., k2): r2 = e2 } : res \ eff
+        // e: e_t \ e_f
+        // ei: e_t \ ei_f
+        // ki: ri -> e_t \ k_f
+        // k_f = (e_f - Eff) + Union_i(ei_f)
+        // res = e_t
+        // eff = k_f
```

Comment:
I think we should either delete these comments or rewrite them in style used for e.g. `Expr.InvokeConstructor2`.